tests/epics/areadetector/test_hdf_streamer_det.py

```diff
@@ -109,18 +122,18 @@ async def test_hdf_streamer_dets_step(
 # TODO: write test where they are in the same stream after
 #   https://github.com/bluesky/bluesky/issues/1558
 async def test_hdf_streamer_dets_fly_different_streams(
-    hdf_streamer_dets: List[HDFStreamerDet], RE, doc_holder: DocHolder
+    hdf_streamer_dets: List[StandardDetector[StandardController, HDFWriter]], RE, doc_holder: DocHolder
```

Comment:
So if we lose genericness we would pass the objects directly in here:
```suggestion
    hdf_streamer_dets: List[Tuple[StandardDetector, ADDriver, NDFileHDF]], RE, doc_holder: DocHolder
```